rdagent/scenarios/data_science/proposal/exp_gen/prompts_v3.yaml

```diff
@@ -0,0 +1,343 @@
+scenario_problem:
+  system: |-
+    {% include "scenarios.data_science.share:scen.role" %}
+    The user is improving a Kaggle competition implementation iteratively. Each new iteration (trace) is typically a modification of the current overall State-of-the-Art (SOTA) solution. If a new trace's performance surpasses the current SOTA, it establishes a new SOTA. Otherwise, it is considered a failed experiment.
+
+    You will be provided with:
+    1. A detailed competition scenario description;
+    2. A history of previous SOTA experiments and their associated feedbacks, typically indexed or ordered from oldest to newest;
+    3. A history of previous failed experiments and their associated feedbacks, chronologically ordered, where each failed experiment did not surpass the SOTA that was current at the time of its execution;
+
+    Your task is to analyze the provided information (primarily the scenario and current SOTA, if available) and identify a concise list of **Key Challenges** or **Core Problems** relevant to achieving success in this competition and improving the target metric. Aim for **FEWER BUT BETTER** challenges (e.g., 2-3 critical challenges), focusing on the most impactful aspects that can be methodically addressed.
+
+    ### Core Analysis Dimensions for Identifying Challenges
+    1. **SOTA Alignment Analysis**: (If SOTA is provided) Systematically compare the current SOTA implementation against dataset properties and domain knowledge to identify discrepancies or areas representing core challenges to overcome for enhancement.
```

Comment:
A question just because of curiosity. This description duplicates with the fields of BaseModel for structure_response. Will it work in practice if we only keep one of them?